### PR TITLE
Update dependency versions

### PR DESCRIPTION
**Pull request by ayeshLK**
## Examples
N/A

## Checklist
- [ ] ~Linked to an issue~
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-version=2.10.1-SNAPSHOT
+version=2.11.0-SNAPSHOT
 puppycrawlCheckstyleVersion=10.12.0
 ballerinaGradlePluginVersion=2.0.1
 
-ballerinaLangVersion=2201.8.0
-stdlibCacheVersion=3.7.0
-stdlibCryptoVersion=2.5.0
+ballerinaLangVersion=2201.9.0-20240405-165800-4b163f78
+stdlibCacheVersion=3.7.1
```
Comment (daneshk) [inline gradle.properties:8]:
```suggestion
stdlibCacheVersion=3.8.0-20240409-101100-e2d7f4e
```

Comment (ayeshLK) [reply gradle.properties]: Fixed with [5344f50b813fe08a9b80d607fac719a51a7eeb43](https://github.com/ballerina-platform/module-ballerina-jwt/pull/1173/commits/5344f50b813fe08a9b80d607fac719a51a7eeb43)

**Comment by daneshk** [comment]
we need to update the distribution version in Ballerina.toml file, if we are going with update 9

File: gradle.properties [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 group=io.ballerina.stdlib
-version=2.10.1-SNAPSHOT
+version=2.11.0-SNAPSHOT
 puppycrawlCheckstyleVersion=10.12.0
 ballerinaGradlePluginVersion=2.0.1
 
-ballerinaLangVersion=2201.8.0
-stdlibCacheVersion=3.7.0
-stdlibCryptoVersion=2.5.0
+ballerinaLangVersion=2201.9.0-20240405-165800-4b163f78
```
Comment (dilanSachi) [inline gradle.properties:7]: no need to update lang

File: gradle.properties [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 stdlibLogVersion=2.9.0
 stdlibTimeVersion=2.4.0
 # Transitive dependencies
 stdlibTaskVersion=2.5.0
-stdlibConstraintVersion=1.4.0
+stdlibConstraintVersion=1.5.0
 stdlibIoVersion=1.6.0
-observeVersion=1.2.0
-observeInternalVersion=1.2.0
+observeVersion=1.2.3-20240403-143400-5f0247b
```
Comment (dilanSachi) [inline gradle.properties:16]: lets update to latest version

File: gradle.properties (hunk 2 above)
Comment (ayeshLK) [reply gradle.properties]: Fixed [6336d15f83936e7dbfc61044b7db645980972f22](https://github.com/ballerina-platform/module-ballerina-jwt/pull/1173/commits/6336d15f83936e7dbfc61044b7db645980972f22)

File: gradle.properties (hunk 3 above)
Comment (ayeshLK) [reply gradle.properties]: Fixed [6336d15f83936e7dbfc61044b7db645980972f22](https://github.com/ballerina-platform/module-ballerina-jwt/pull/1173/commits/6336d15f83936e7dbfc61044b7db645980972f22)

**Review by dilanSachi — APPROVED** [review]

**Pull request merged**